src/classes/Bot.ts

```diff
@@ -34,11 +34,10 @@ class Bot {
 		password: process.env.MINECRAFT_PASSWORD,
 		host: "mc.hypixel.net",
 		auth: "microsoft",
-		version: "1.16.4",
+		version: "1.8.9",
```

Comment:
Why have you changed the version? AFAIK, this only reduces our chat character limit from 256 -> 100 (changed in 1.12)